src/form/Form.tsx

```diff
@@ -1053,29 +1056,88 @@ function Form({
     }
   };
 
-  function handleActions(setLoader: any) {
-    for (let i = 0; i < activeStep.servar_fields.length; i++) {
-      const servar = activeStep.servar_fields[i].servar;
-      const fieldVal = fieldValues[servar.key];
-      if (servar.type === 'login') {
-        setLoader();
-        // Unless we want to do something more complex here we need to make a
-        // choice and can't just return both. Prioritize stytch
-        // @ts-expect-error
-        if (integrations.stytch) {
-          return sendMagicLink(fieldVal);
-        } else {
-          return sendLoginCode(fieldVal, servar);
+  // usePayments /Stripe
+  const [setCardElement, setupPaymentMethod] = usePayments();
```

Comment:
i think it would be cleaner to pull setupPaymentMethod out of the hook, then you can pull the entire actionConfigurations out into a global const too.

Reply:
i might put it in integrations/utils

Reply:
One alternative that I just that thought of is to move targetElementFn out of actionConfigurations and then in handleActions always just set a function called getCardElement into the ActionData (but only stripe would ever use it).  Not perfect, but I think actionConfigurations could then be a static global.  I favor leaving it as is, but if you like this idea  or some other idea better let me know.